rest-client/src/main/java/org/sagebionetworks/bridge/rest/UserSessionInfoProvider.java

```diff
@@ -22,17 +22,33 @@
     private static final Logger LOG = LoggerFactory.getLogger(UserSessionInfoProvider.class);
 
     private final AuthenticationApi authenticationApi;
-    private final SignIn signIn;
+    private final String studyId;
+    private final String email;
+    private final String password;
     private UserSessionInfo session;
 
     UserSessionInfoProvider(AuthenticationApi authenticationApi, SignIn signIn) {
```

Comment:
Might be good for this to take an optional UserSessionInfo as well, accompanied with the ability to initialize ApiClientProvider with SignIn+UserSessionInfo pairs